### ISSUE TITLE
Prevent other users from provisioning servers when accessing puppetmaster-gui

### ISSUE THREAD

**Issue by nbeernink** [atomia/puppetmaster-gui#52]
I understood from the readme that each provision step should be done in order. However, the installation interface currently doesn't prevent other users from accessing the URL and allows them to start provisioning servers, then when the other person proceeds the wrong server can be provisioned. 

For example: user 1 starts provisioning the fsagent server while user 2 starts provisioning the installatron server. When you time it correctly the fsagent server can be provisioned with the installatron setup. 

Perhaps the sessions can be locked so that session hijacking isn't possible?

**Comment by stefanmortensen** [comment]
Yes good point, haven't really taken the multi user aspect into account yet.
